argilla-server/src/argilla_server/api/handlers/v1/datasets/records.py

```diff
@@ -358,6 +364,8 @@ async def search_current_user_dataset_records(
             query_score=record_id_score_map[record.id]["query_score"],
         )
 
+    await telemetry_client.track_crud_records(action="read", record_or_dataset=dataset, count=search_responses.total)
```

Comment:
Again, I would prefer to rename the action to "search" instead of "read", since we already have a "read" records endpoint. Also, adding the "me" context would be nice to separate contextualized searches (normally performed from the UI) from common searches ( normally performed from the SDK). 
```suggestion
    await telemetry_client.track_crud_records(action="/me/search", record_or_dataset=dataset, count=search_responses.total)
```

Reply:
Sure, I will review this bahavior.